torch/_functorch/autograd_function.py

```diff
@@ -272,3 +273,45 @@ def __setattr__(self, name, value):
             self.__dict__[name] = value
             return
         return setattr(self._pt_inner_ctx, name, value)
+
+
+def reductify_leaf(grad_input, grad_input_bdim, input_bdim, input_shape_without_bdim, batch_size):
```

Comment:
The document has `reductify`, which is a more general version of `reductify_leaf`. The signature of reductify depends on the next PR, so this PR just introduces `reductify_leaf`.